### ISSUE TITLE
Pi0.5 reproduction

### ISSUE THREAD

**Issue by SMLee0407** [Physical-Intelligence/openpi#973]
Hello,

I’m trying to fine-tune Pi0.5 on LIBERO or RoboCasa, but it seems that this repo only provides a pretrained Pi0.5 checkpoint and does not include subtask prediction.

Is it typical to reproduce Pi0.5 using the same fine-tuning process as Pi0, with only the continuous action prediction loss for the action expert and VLM?

In other words, is the checkpoint path the only difference between reproducing Pi0 and Pi0.5?

Thanks.
